app/views/course/assessment/submission/submissions/_question.json.jbuilder

```diff
@@ -17,4 +17,9 @@ json.type case question.actable_type
             'Scribing'
           end
 
+if question.specific.is_a?(Course::Assessment::Question::Programming)
```

Comment:
Are we supporting this for all programming languages?

Reply:
With this PR, `course_assessment_question_programming` will have a boolean field `file_submission` that will indicate whether it uses this or not, by default other programming languages will have this set to false, but if we wish to port this over to use with other languages, then we can set `file_submission` to true when the package is created, and the front-end should accommodate it.